how-to-contribute.md

```diff
@@ -0,0 +1,65 @@
+# How to contribute to the Knowledge Hub
+
+We are building the Knowledge Hub in public and we would like to encourage contributions from the community. 
+
+The only thing you need to contribute to the Knowledge Hub is a Github account. You can sign up to GitHub by filling out the form [on this page](https://github.com/join).
+
+Got it? Well done! You can now contribute to the Knowledge Hub in one of the following ways:
+
+## Talk to us
+
+The Knowledge Hub project on Github has its own forum. This forum can be used by anyone to: 
+
+* report and discuss problems
+* propose improvements and engage in discussions about improvements proposed by other contributors
+* present ideas and discuss ideas presented by other contributors
+
+For example:
+
+Alice wants to report a mistake in one of the articles. She can do this by filling up the form [here](https://github.com/alixedi/knowledge-hub/issues/new). This will create a new issue and she will be able to discuss the problem with the community and even offer suggestions about how to solve it.
```

Comment:
**To report a mistake in existing content**: submit a [new issue](https://github.com/alixedi/knowledge-hub/issues/new). Once the issue has been created, it automatically becomes a topic for discussion in the forum